pkg/sql2pgroll/convert.go

```diff
@@ -4,26 +4,24 @@ package sql2pgroll
 
 import (
 	"fmt"
+	"time"
 
 	pgq "github.com/xataio/pg_query_go/v6"
 
 	"github.com/xataio/pgroll/pkg/migrations"
 )
 
-var ErrStatementCount = fmt.Errorf("expected exactly one statement")
-
-// Convert converts a SQL statement to a slice of pgroll operations.
-func Convert(sql string) (migrations.Operations, error) {
+// Convert converts SQL statements to a pgroll migration.
+func Convert(sql string) (migrations.Migration, error) {
 	ops, err := convert(sql)
 	if err != nil {
-		return nil, err
-	}
-
-	if ops == nil {
-		return makeRawSQLOperation(sql), nil
+		return migrations.Migration{}, err
 	}
 
-	return ops, nil
+	return migrations.Migration{
+		Name:       generateUniqueMigrationName(),
```

Comment:
I choose the current timestamp as the migration name to make migration ordering easier.